src/engraving/tests/tst_note.cpp

```diff
@@ -513,6 +514,18 @@ void TestNote::tpcDegrees()
     //QCOMPARE(tpc2degree(Tpc::TPC_B_S, Key::C_S), 7);
 }
 
+void TestNote::alteredUnison()
+{
+    MasterScore* score = readScore(NOTE_DATA_DIR + "altered-unison.mscx");
+    Measure* m = score->firstMeasure();
+    Chord* c = m->findChord(Fraction(0, 1), 0);
+    QVERIFY(c->downNote()->accidental() && c->downNote()->accidental()->accidentalType() == Ms::AccidentalType::FLAT);
+    QVERIFY(c->upNote()->accidental() && c->upNote()->accidental()->accidentalType() == Ms::AccidentalType::NATURAL);
```

Comment:
Would have preferred to use QCOMPARE however this is crashing for me using MSVC debug C runtime, when it tries to `delete` the strings returned by QTest::toString( ). No idea why, it all looks fine stepping through it.